sonar-plugin/src/main/java/io/ecocode/javascript/JavaScriptRulesDefinition.java

```diff
@@ -0,0 +1,41 @@
+package io.ecocode.javascript;
+
+import org.sonar.api.SonarRuntime;
+import org.sonar.api.server.rule.RulesDefinition;
+import org.sonarsource.analyzer.commons.RuleMetadataLoader;
+
+import java.util.Collections;
+
+public class JavaScriptRulesDefinition implements RulesDefinition {
+
+    private static final String METADATA_LOCATION = "io/ecocode/javascript/rules/javascript";
```

Comment:
To facilitate the future integration of rules from the shared rules repository (cf. PR green-code-initiative/ecoCode#178), I advise you to put the HTML and JSON resources in the folder:
`sonar-plugin/src/main/resources/io/ecocode/rules/js/`

```suggestion
    private static final String METADATA_LOCATION = "io/ecocode/rules/js";
```

Reply:
Oh yes I understand, so I propose to use `io/ecocode/rules/javascript` (I prefer this as "js" because its more explicit)

Reply:
> Oh yes I understand, so I propose to use `io/ecocode/rules/javascript` (I prefer this name as "js" because its more explicit)

I relocate specification in corresponding folder in following : https://github.com/green-code-initiative/ecoCode/pull/209